tests/publiccloud/run_ltp.pm

```diff
@@ -90,7 +130,20 @@ sub run {
         $instance->run_ssh_command(cmd => 'sudo zypper -q in -y ltp',                              timeout => 600);
     }
 
-    $ver_linux_log = $instance->run_ssh_command(cmd => get_ltproot() . "/ver_linux");
+    download_whitelist();
+    my $ltp_env = gen_ltp_env($instance);
+    $self->{ltp_env} = $ltp_env;
+
+    # Use lib/LTP/WhiteList module to exclude tests
+    if (get_var('LTP_KNOWN_ISSUES')) {
+        my $exclude       = get_required_var('COMMAND_EXCLUDE');
+        my @skipped_tests = list_skipped_tests($ltp_env, get_required_var('COMMAND_FILE'));
+        if (@skipped_tests) {
+            $exclude .= '|' if (length($exclude) > 0);
```

Comment:
```suggestion
            $exclude .= '|' if (length($exclude) > 1);
```

Reply:
not sure about this. Why you want to be sure that the $exclude string is at least 2 chars long?

Reply:
I assume that this line is adding the `|` separator but it is needed only when there are at least two things to be separated.

Reply:
`$exclude` is a string (regex). `|` needs to be added when the current string is not empty.